plugins/automation/spec/system/page_objects/discourse_automation/automation.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class Automation < PageObjects::Pages::Base
+      def visit(automation)
+        super("/admin/plugins/discourse-automation/#{automation.id}")
+        self
+      end
+
+      def set_name(name)
+        form.find('input[name="automation-name"]').set("aaaaa")
+        self
+      end
+
+      def has_error?(message)
+        form.has_content?(message)
+      end
+
+      def has_name?(name)
+        form.find_field("automation-name").value == name
+      end
+
+      def set_triggerables(triggerable)
+        select_kit = PageObjects::Components::SelectKit.new(".triggerables")
+        select_kit.expand
+        select_kit.select_row_by_value(triggerable)
+        self
+      end
+
+      def update
+        form.find(".update-automation").click
+        form.has_selector?(".update-automation.is-loading")
```

Comment:
This might end up being flaky if the server running the test is slow as it may not see the loading indicator. Is there a success message we can check for after we update the automation? That might make things less flaky.

Reply:
Yes Im not sure here. The thing is what is flaky atm is that we don't have the error message. I'm not sure of why at the moment, my only guess is that sometimes it takes more time to make the update and show the message. That's surprising but I don't have any other clue about this for now.

Reply:
I guess I will merge and see how it goes thanks